stablehlo/dialect/TypeInference.cpp

```diff
@@ -4054,6 +4202,13 @@ LogicalResult verifyReshapeOp(std::optional<Location> location, Value operand,
                              ") doesn't match expected number of elements (",
                              numOperandElements, ")");
 
+  // reshape_c1
+  // part of reshape_c1 already verified by
+  // Trait HLO_CompatibleOperandsAndResultElementType
+  if (failed(sameQuantScaleZeroPoint(location, operand.getType(),
```

Comment:
should be addressed via #2112

Reply:
we will need this check in absence of the new Trait.